slipp/src/test/java/slipp/DispatcherServletTest.java

```diff
@@ -29,7 +29,7 @@ void setUp() throws Exception {
         DatabasePopulatorUtils.execute(populator, ConnectionManager.getDataSource());
 
         dispatcher = new DispatcherServlet();
-        dispatcher.addHandlerMpping(new AnnotationHandlerMapping("slipp.controller"));
+        dispatcher.addHandlerMapping(new AnnotationHandlerMapping("slipp.controller"));
```

Comment:
```suggestion
        dispatcher.addHandlerMapping(new AnnotationHandlerMapping(new BeanScanner("slipp")));
```
위와 같이 변경되어야 테스트가 통과하지 않을까요?
코드를 변경 후 테스트로 확인하는 습관을 들이시면 추후에도 큰 도움이 될 것 같습니다 😊

Reply:
테스트를 확인하지 못했네요! 수정하였습니다 감사합니다 :)